.github/workflows/build-and-push-components.yml

```
@@ -22,6 +22,35 @@ on:
         type: string
         description: 'Name of the Docker Bake group of targets'
         default: default
+  workflow_dispatch:
+    inputs:
+      git_ref:
+        required: false
+        type: string
+        description: 'The branch, tag or SHA to build.'
+        default: ''
+      image_tag:
+        required: true
+        type: string
+        description: 'Image tag to use.'
+      registry_name:
+        required: false
+        type: string
+        description: 'Name of the registry.'
+        default: ghcr.io/openclarity
+      suffix:
+        required: false
+        type: string
+        description: 'Suffix to append to the image name'
+        default: ""
```

```suggestion
        default: "-dev"
```

I think the default could still be `-dev`

---

I worked based on the previous version, we did not have suffix there: https://github.com/openclarity/vmclarity/pull/1768/files

But I am not sure why this suffix was added here.

---

https://github.com/openclarity/vmclarity/pull/1848#discussion_r1656824040

---

Thanks, the dev suffix has been added.